enterprise/web.jsf.editor/src/org/netbeans/modules/web/jsf/editor/completion/JsfAttributesCompletionHelper.java

```diff
@@ -246,15 +247,15 @@ private static void addTypesFromPackages(CompletionContext context, CompilationC
         }
     }
 
-    public static void completeSectionsOfTemplate(final CompletionContext context, final List<CompletionItem> items, String ns, OpenTag openTag) {
+    public static void completeSectionsOfTemplate(final CompletionContext context, final List<CompletionItem> items, String ns, OpenTag openTag, JsfVersion jsfVersion) {
```

Comment:
See my comment in `JsfUtils.getRoot`: I think the final parameter is not necessary.